ui/templates/welcome.html

```diff
@@ -3,7 +3,11 @@
     <div class="section-header">
         <div class="row">
             <div class="col-md-2 col-md-offset-5">{% if perms.learningresources.add_repository %}<a href="{% url 'create_repo' %}" role="button"
-                                                     class="btn btn-primary btn-large">Create repository</a>{% endif %}</div>
+                                                     class="btn btn-primary btn-large">Create repository</a>
```

Comment:
It looked weird to me, too, but I just added to what was there.  I'll change it.